package/src/renderer/Canvas.tsx

```diff
@@ -122,6 +122,7 @@ export const Canvas = forwardRef<SkiaView, CanvasProps>(
     // Draw callback
     const onDraw = useDrawCallback(
       (canvas, info) => {
+        canvas.clear(0);
```

Comment:
how about making this configurable via a boolean property? default will clear the canvas, but the property would allow the user to opt out of that behavior.

Reply:
we can definitely do that, I was wondering if not clearing make sense in the context of a declarative API. @chrfalch?

Reply:
I think this is an option that will very seldom be used in the context of a Canvas, so I would suggest we go without adding this to our API and recommend to use the declarative API if you want this kind of control.